wondrous-app/utils/constants.ts

```diff
@@ -65,6 +65,7 @@ export const PAYMENT_STATUS = {
 // Supported Chains (ETHEREUM, POLYGON)
 const SUPPORTED_CHAINS = {
   1: 'ETH',
+  3: 'ETH', // REMOVE BEFORE COMMIT
```

Comment:
Should this be removed?

Reply:
was a placeholder to get proposal to work with my test snapshot space, which was on Ropsten -- removed